app/assets/stylesheets/components/_contents-list-with-body.scss

```diff
@@ -1,3 +1,11 @@
+.app-c-contents-list-with-body .app-c-contents-list {
```

Comment:
There's a bit of negotiating the existing implementation here, if we set an isolated style it would affect all content lists (which at the moment would impact unconverted formats).
I'd expect this rule to become an isolated component rule once all formats with content lists are moved to the new layout. (My firebreak is concentrating on tech-debt issues like this as we complete the design changes).
That said, if there's another way to achieve this I'm happy to update.

Reply:
There's a pattern for modifying margins on components with margin parameters.

Reply:
I've removed the commit that introduced those styles and added
https://github.com/alphagov/government-frontend/pull/659/commits/f69c8a72bc9d76b76a813655b6befc43fe8b6e46
which seems less smelly.